Kernel/FileSystem/ProcFS.cpp

```diff
@@ -372,6 +374,16 @@ Optional<KBuffer> procfs$interrupts(InodeIdentifier)
     return builder.build();
 }
 
+Optional<KBuffer> procfs$keymap(InodeIdentifier)
+{
+    KBufferBuilder builder;
+    JsonObjectSerializer<KBufferBuilder> json { builder };
+    auto name = KeyboardDevice::the().keymap_name();
+    json.add("keymap", name);
```

Comment:
I would lose the temporary here:
```suggestion
    json.add("keymap", KeyboardDevice::the().keymap_name());
```